File: src/codes/Users_Manual/stm/gain_2.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
auto stm = autd3::GainSTM::from_freq(1.0 * autd3::Hz)
               .with_mode(autd3::GainSTMMode::PhaseFull);
//~return 0; }

File: src/codes/Users_Manual/stm/gain_2.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using static AUTD3Sharp.Units;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
var stm = GainSTM.FromFreq(1.0f * Hz, [new Null(), new Null()]).WithMode(GainSTMMode.PhaseFull);